app/v2/controllers/requestParsers/validators/AmendForeignPropertyAnnualSubmissionValidator.scala

```diff
@@ -44,21 +44,7 @@ class AmendForeignPropertyAnnualSubmissionValidator @Inject()(appConfig: AppConf
     }
 
   private def bodyFormatValidation: AmendForeignPropertyAnnualSubmissionRawData => List[List[MtdError]] = { data =>
-    val baseValidation = List(
-      JsonFormatValidation.validate[AmendForeignPropertyAnnualSubmissionRequestBody](data.body))
-
-    val extraValidation: List[List[MtdError]] = {
-      data.body
-        .asOpt[AmendForeignPropertyAnnualSubmissionRequestBody]
-        .map(_.isEmpty)
-        .map {
-          case true  => List(List(RuleIncorrectOrEmptyBodyError))
-          case false => NoValidationErrors
-        }
-        .getOrElse(NoValidationErrors)
-    }
-
-    baseValidation ++ extraValidation
+    ???
```

Comment:
I'm unsure why this has been changes, could you clarify that with me?

Reply:
Only because it did not compile and the validators had not yet been done. I also removed the validator spec so that this code is never used - but I should also have removed the controller from the route: just doing that now.

I could have deleted the file but then I would have needed to delete everything that uses it (which are useful starting points for development).